v2/stacks/auth/.env

```diff
@@ -14,7 +14,8 @@ PATH_MANIFESTS="../../manifests"
 PATH_PROVISIONING="../../provisioning"
 
 # -- Global variable overrides
-IS_PUBLISHING="Y"
+IS_PUBLISHING=true
```

Comment:
I'm fairly sure Babbage uses Y/N to do feature flags, so why are you changing this?

Reply:
You're right, but alas, the files-api, image-api and download-service (and perhaps others) all use https://pkg.go.dev/strconv#ParseBool for this var

We will have to make an exception for babbage.

Reply:
ah very good, ok then.